src/modm/platform/core/cortex/delay.cpp.in

```diff
@@ -26,9 +26,9 @@
 %# FIXME: Move out of this modules, maybe remove completely
 %# TODO: These numbers were measured from a few example applications.
 %#       A statistical analysis of the average overhead cycles is required!
-%% if partname.startswith("stm32f4")
+%% if target.family in ["f4"]
```

Comment:
What happens when Atmel announces a SAM***F4*** series?

Reply:
There shouldn't be any STM32 specific stuff in the Cortex-M module anyways. Hence the fixme's ;-P